content/about.hbs

```diff
@@ -85,54 +85,12 @@
 		<div class="container">
 			<h1>Testimonials</h1>
 
-			<article class="media testimonial">

```

Comment:
Since none of these testimonials are obviously out of date (to anyone outside of the detachment), I'd recommend keeping them in place until we get replacements so the website looks less bare